### ISSUE TITLE
Generic dialog 1 - Rescheduling activity 

### ISSUE THREAD

**Issue by KristellP** [PerfectFit-project/content-development#12]
This dialog is meant to ask whether the user wants to/can do an activity now or rather later. 

To do: 
- [x] Have dialog checked by Nele or Milon
- [x] Put dialog in a ppt slide to present to Anke tomorrow
- [ ] Check with base requirements

**Comment by nelealbers** [comment]
@KristellP @svenvanderburg  We wrote a couple comments in the Excel file. Milon corrected the Dutch and made it simpler. It is important that we keep in mind to make the sentences as short and simple as possible for our target group. We also added a few questions that maybe should form part of the timing issue. They evolve around how many times people can choose "later," and what we do with people who repeatedly fail to do a dialog (especially if later dialogs are building on this dialog).
